plugins/UM3NetworkPrinting/src/Cloud/CloudOutputDeviceManager.py

```diff
@@ -31,6 +32,7 @@ class CloudOutputDeviceManager:
     META_CLUSTER_ID = "um_cloud_cluster_id"
     META_NETWORK_KEY = "um_network_key"
     SYNC_SERVICE_NAME = "CloudOutputDeviceManager"
+    META_LINKED_TO_ACCOUNT = "linked_to_account"
```

Comment:
This should be renamed, since it's not clear what this means outside of this context. I think something like "added_by_um_account" or "um_linked_to_account" is better.